packages/mobile/src/screens/collection-screen/CollectionScreen.tsx

```diff
@@ -156,6 +157,8 @@ const CollectionScreenComponent = (props: CollectionScreenComponentProps) => {
 
   const { neutralLight5 } = useThemePalette()
 
+  const releaseDate =
+    'release_date' in collection ? collection.release_date : created_at
```

Comment:
nit: does `collection?.release_date ?? created_at` work?

Reply:
no, sadly we need the type narrowing